modin/engines/ray/cudf_on_ray/frame/axis_partition.py

```diff
@@ -11,13 +11,24 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""Module houses classes of axis partitions implemented using Ray and cuDF."""
+
 import cudf
 import ray
 
 from .partition import cuDFOnRayFramePartition
 
 
 class cuDFOnRayFrameAxisPartition(object):
+    """
+    Base class for any axis partition class based on ``cuDFOnRayFramePartition``.
```

Comment:
I do not understand this phrase. What does this mean `based on cuDFOnRayFramePartition`?

Reply:
rephrased